private/scripts/helpers.sh

```diff
@@ -460,8 +467,14 @@ function clean_up() {
     exit 1;
   fi
 
+  is_multisite=$(terminus wp -- "$sitename"."$siteenv" config get MULTISITE)
```

Comment:
Prefer `config is-true`
https://developer.wordpress.org/cli/commands/config/is-true/